org.eclipse.nebula.widgets.nattable.extension.nebula.feature/feature.xml

```diff
@@ -19,28 +19,16 @@
       %license
    </license>
 
-   <plugin
-         id="org.eclipse.nebula.widgets.nattable.extension.nebula"
-         version="0.0.0"/>
-
-   <plugin
-         id="org.eclipse.nebula.widgets.richtext"
-         version="0.0.0"/>
-
-   <plugin
-         id="org.eclipse.nebula.widgets.cdatetime"
-         version="0.0.0"/>
-
-   <plugin
-         id="org.eclipse.nebula.cwt"
-         version="0.0.0"/>
+   <requires>
+      <import plugin="org.apache.commons.lang3"/>
+      <import plugin="org.apache.commons.text"/>
+      <import plugin="org.eclipse.nebula.cwt"/>
+      <import plugin="org.eclipse.nebula.widgets.cdatetime"/>
+      <import plugin="org.eclipse.nebula.widgets.richtext"/>
+   </requires>
```

Comment:
```suggestion
```

These are also likely not independently required?

Reply:
Yes, less is more.